src/QMCHamiltonians/SkAllEstimator.cpp

```diff
@@ -145,12 +144,12 @@ SkAllEstimator::Return_t SkAllEstimator::evaluate(ParticleSet& P)
       values[NumK + 2 * k + 1] = w * RhokTot_i[k];
     }
   }
-#else
+#else // Test case only touches this path at the moment
```

Comment:
We always use macro USE_REAL_STRUCT_FACTOR. Why the test case touch #else?

Reply:
Fixed.